python/ql/test/query-tests/Expressions/general/UseofApply.expected

```diff
@@ -0,0 +1 @@
+| expressions_test.py:270:3:270:17 | ControlFlowNode for apply() | Call to the obsolete builtin function 'apply'. |
```

Comment:
You'll need to create two copies of this file (and the `.qlref` file), and put them in the `2` and `3` directories. This is because `apply` is no longer available as a built-in in Python 3.

(We could also just have one copy of the tests, in the `2` directory, but I guess it's good to have the check that the local version of `apply` isn't caught in both versions.)

Also, maybe just put the test case in a separate `.py` file, say, `use_of_apply.py`. Having these huge monolithic test files can be a bit cumbersome.

Reply:
Ah I see!

Reply:
Regarding huge monolithic test files, strong agree. I was contemplating making a single test case file for all of the different queries I'm modernizing but figured it'd be better to stick with `expressions_test.py` for now to avoid too much noise.